src/las/header.cpp

```diff
@@ -41,6 +41,7 @@ LasHeader LasHeader::FromLazPerf(const lazperf::header14 &header)
     h.min.y = header.miny;
     h.max.z = header.maxz;
     h.min.z = header.minz;
+    h.span = h.ComputeSpan();
```

Comment:
As in ```static double ComputeSpan(Vector3 min, Vector3 max)```? What's the advantage of that?

Reply:
it's just redundant to have span and ComputeSpan do the same thing, it encourages `span` to get out of date

Reply:
My preference would be to remove the `span` property and do the function call every time, since span isn't a property of the lasheader itself

Reply:
> it's just redundant to have span and ComputeSpan do the same thing, it encourages `span` to get out of date
I agree with that 

> My preference would be to remove the `span` property and do the function call every time, since span isn't a property of the lasheader itself
I feel like at this point LasHeader is mostly a helper for copc, so if we can add an argument to avoid computing span everytime we should do it, no?

Reply:
If you want to keep the property, maybe let's name it `max_bounds_span` or something? And make ComputeSpan a static function (maybe even a private static function, or move it to a utility class?)

Reply:
I've updated ComputeScan to not be redundant

Reply:
> If you want to keep the property, maybe let's name it `max_bounds_span` or something? And make ComputeSpan a static function (maybe even a private static function, or move it to a utility class?)

Ok I'll do that in a bit

Reply:
I'm actually removing it in the end